client/lib/signup/step-actions/index.js

```diff
@@ -534,6 +534,40 @@ export function createSite( callback, dependencies, stepData, reduxStore ) {
 	} );
 }
 
+export function createWpForTeamsSite( callback, dependencies, stepData, reduxStore ) {
+	const { site, siteTitle } = stepData;
+	const state = reduxStore.getState();
+
+	// The new p2 theme for WP for Teams project.
+	// More info: https://wp.me/p9lV3a-1dM-p2
+	const themeSlugWithRepo = 'pub/p2020';
+
+	const data = {
+		blog_name: site,
+		blog_title: siteTitle,
+		public: getNewSitePublicSetting( state ),
+		options: { theme: themeSlugWithRepo, timezone_string: guessTimezone() },
+		validate: false,
+	};
+
+	wpcom.undocumented().sitesNew( data, function( errors, response ) {
+		let providedDependencies, siteSlug;
+
+		if ( response && response.blog_details ) {
+			const parsedBlogURL = parseURL( response.blog_details.url );
+			siteSlug = parsedBlogURL.hostname;
+
+			providedDependencies = { siteSlug };
+		}
+
+		if ( user.get() && isEmpty( errors ) ) {
+			fetchSitesAndUser( siteSlug, () => callback( undefined, providedDependencies ), reduxStore );
+		} else {
+			callback( isEmpty( errors ) ? undefined : [ errors ], providedDependencies );
+		}
+	} );
+}
```

Comment:
Will leave it up to you, but it might be worth seeing if there are any straightforward ways of reducing duplication between your new `createWpForTeamsSite` and `createSite`.  If it's feeling awkward/complex, feel free to skip.

Reply:
Even though it looks pretty much the same now, I would keep it separate. We have some custom stuff in there and might need more of it in the future.